proto/cosmos/staking/query.proto

```diff
@@ -0,0 +1,176 @@
+syntax = "proto3";
+package cosmos.staking;
+
+import "cosmos/query/pagination.proto";
+import "gogoproto/gogo.proto";
+import "cosmos/staking/staking.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/staking/types";
+
+// Query provides defines the gRPC querier service
+service Query {
+//  rpc Validators (QueryValidatorsRequest) returns (QueryValidatorsResponse) {}
+
+  rpc ValidatorQ (QueryValidatorRequest) returns (QueryValidatorResponse) {}
```

Comment:
Why ValidatorQ? why not just Validator?

Reply:
There's another keeper method as Validator, I used this as I wasn't sure on what would be better.

Reply:
We will remove that and use grpc query right?

Reply:
No, for now the plan is to have both querier and grpc for v0.39

Reply:
Protobuf/gRPC RPC/type names take precedence over keeper names. As an alternative we can wrap the keeper in another struct to implement the QueryServer. But we should never make go names more important than protobuf